docs/.vuepress/theme/styles/theme-dark.styl

```diff
@@ -197,11 +197,23 @@ html.theme-dark {
       border-bottom-color $tabsComponentBorderColor
       background $tabsComponentBackgroundColor
 
-      a {
-        color $fontColor !important
+      .tabs-component-tab {
+        a {
```

Comment:
It could be chnged to `a.tabs-component-tab`. It will be probably more clear.

Reply:
`a` tag is the child tag of `li` `.tabs-component-tab`. And whenever a tab is disabled the `li` `.tabs-component-tab` is getting a CSS class `is-disabled`. Then, the styling of the `a` tag depends on the parent `li` tag.

Reply:
Ah, yeah. Sorry!

Reply:
@Osky772 Maybe use `.tabs-component-tab a`?